### PR TITLE
Implement cron trigger

### PR DESCRIPTION
**Pull request by shinsuke-nara**
Related issue No. 830
Release version 0.12.0

### CODE REVIEW

File: thingif/src/androidTest/java/com/kii/thingif/ThingIFAPI_PostNewTriggerForSchedulePredicateTest.java [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Created on 2016/05/17.
+ */
```
Comment (nfukuzaki) [inline thingif/src/androidTest/java/com/kii/thingif/ThingIFAPI_PostNewTriggerForSchedulePredicateTest.java]: unnecessary comment

**Comment by nfukuzaki** [comment]
Can you add the large test?

**Comment by shinsuke-nara** [comment]
https://github.com/KiiPlatform/thing-if-AndroidSDK/pull/86#discussion_r63491362

I removed redundant comment.

**Comment by shinsuke-nara** [comment]
@nfukuzaki Larget test is added. Could you review this?

**Comment by nfukuzaki** [comment]
LGTM

**Pull request merged**